source/brsHamcrest_Matchers/brsHamcrest_CoreMatchers.brs

```diff
@@ -15,6 +15,26 @@ function is (matcher as Object) as Object
 end function
 
 
+'A matcher that has the correct structure, but returns true to anything, for the purposes of integration with frameworks
+'Is capable of handling an optional parameter
+'
+'Example:
+'assertThat(foo, isAnything())
+'
+'@return {Object<Matcher>} A Matcher
+function isAnything (optionalValue="" as Dynamic) as Object
```

Comment:
Not sure what my thinking was at the time, perhaps I was just wanted to be as open as possible, I'll remove it.